project.clj

```diff
@@ -6,14 +6,18 @@
   :uberjar-name "bulk-typer-standalone.jar"
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [me.raynes/fs "1.4.6"]
+                 [com.novemberain/langohr "3.6.1"]
+                 [org.clojure/math.numeric-tower "0.0.4"]
                  [org.cyverse/common-cli "2.8.1"]
+                 [org.cyverse/clj-icat-direct "2.8.7-SNAPSHOT"]
```

Comment:
I'll do a release of clj-icat-direct at some point before merge. Not doing it yet in case there's some useful changes to be made.